projects/dev-schematics/README.md

```diff
@@ -0,0 +1,54 @@
+# Getting Started
+
+Spartacus dev-schematics is a set of collections including pre-defined configurations, like context with base sites or routing.
+
+## Prerequisites
+
+For dev-schematics to be applied properly, you need an Angular application with Spartacus installed.
+
+### Prerequisites for local testing
+
+There might be a situation where you want to use the `@spartacus/schematics` that hasn't been released yet (e.g. its public API is not up-to-date).
+
+These are the steps to take in order to test both `@spartacus/schematics` and `spartacus/dev-schematics` locally:
+
+1. Launch local verdaccio in yours terminal with command `verdaccio`
+2. **(Optional step)**: From the project root go to `/projects/schematics` and change lib version in package.json file (for local testing purposes only, please remember to revert it before committing any changes)
+3. Now (in new terminal or tab) go to dev-schematics scripts: `/projects/dev-schematics/scripts`
+4. Run `./test-dev-schematics-local.sh <version>` where `<version>` is **optional** argument for providing schematics version if it was changed in optional step (`projects/schematics/package.json`). The script will perform all necessary steps to ensure up to date schematics version on your local verdaccio. When the script finishes running, you should be ready to test your dev-schematics.
+
+## Adding dev-schematics to your Spartacus project
+
+Make sure that you have `.npmrc` in the root of your testing project, and that it contains the following content: `@spartacus:registry=http://localhost:4873`.
+
+Run the following command from dev-schematics project:
+
+`ng add @spartacus/dev-schematics --baseSites --routing --outlets`
+
+> NOTE: Please be aware that dev-schematics without params is an empty shell that doesn't perform any actions.
+
+### Available options
+
+Options are provided as flags for main install command as `--<option>`:
+
+- `baseSites`: Adds pre-defined base site list to the app module
+- `routing`: Adds pre-defined routing to the app module
+- `outlets`: Adds testing outlets module to app module. **NOTE:** If you plan to use testing outlets module, please remember to run `outlet-template-files.sh` bash script beforehand, as it will ensure up to date testing module files to be applied.
```

Comment:
Maybe re-phrase this to sound like a note, instead of a requirement.
As this is done under the hood, it might confuse some people; it would be nice to mention this as well.